### PR TITLE
Arc admin attribute form updates

### PR DESCRIPTION
**Pull request by kbrownomni**
For Arcgis plugin: Put admin, events and layers each in cards. Pulled admin attributes out of dialog and into a reactive form.

### CODE REVIEW

**Comment by Skosche3** [comment]
Add visual to reference
image
image

**Comment by Skosche3** [comment]
@newmanw made some changes to the UI that will need to be accounted for.
See "Add Feature Service" section
image
image

**Review by Skosche3 — CHANGES_REQUESTED** [review]
Some initial comments. I can take a look again once you catch up to latest changes on develop/

File: plugins/arcgis/web-app/projects/main/src/lib/arc-admin/arc-admin.component.html [hunk 1]
```diff
@@ -1,188 +1,181 @@
...
+    <mat-card appearance="outlined">
+      <mat-card-header>
+        <mat-card-title>Attributes</mat-card-title>
+        <mat-card-subtitle>MAGE Field to ArcGIS Attribute mappings.</mat-card-subtitle>
+      </mat-card-header>
+      <mat-card-content>
+        <form [formGroup]="attributesForm">
+          <div class="edit-attributes-form">
+            <div class="observation-id-field">
+              <mat-form-field [style.width.%]="90" appearance="fill">
```
[the remark this answers is not in the document]
Comment (kbrownomni) [reply plugins/arcgis/web-app/projects/main/src/lib/arc-admin/arc-admin.component.html:86]: I modeled this 90% width after the sfts plugin UI. I'll check with Billy and see how he'd like it to look.

**Pull request closed without merging**